### DISCUSSION TITLE
scrollspy

### DISCUSSION THREAD

**Discussion by jpg13150** [templaza/astroid-framework#878] [Q&A]
Hello
I use the framwok astroid with template two
I'm trying to create an article with the scrollpsy component 
I selected the corresponding js
I integrated the code from the sample documentation from the Bootstrap documentation https://getbootstrap.com/docs/5.0/components/scrollspy/
I've tried several times but I can't configure correctly. 
When I click on one of the buttons, the whole article moves upwards and a part whose navigation bar disappears under the header of the site
Do you have an idea to solve this problem
Thank you

**Comment by jpg13150** [comment]
Hello

Attached is the address of my test site

[https://jpdetarascon.cloudaccess.host/i ... /scrollspy](https://jpdetarascon.cloudaccess.host/index.php/scrollspy)

Ideally, the article would scroll to the navbar and then scroll below

Thank you for your help

**Comment by sonvnn** [comment]
You have to enable ScrollSpy script to use it. Go to Template Options -> Bootstrap Settings -> Bootstrap JS Libraries -> ScrollSpy
![Screen Shot 2024-11-25 at 08 41 16](https://github.com/user-attachments/assets/42954406-f171-47b0-a30c-9c0897958319)

**Comment by jpg13150** [comment]
Hi
Yes the bootstrap librarie is selected
but it doesn't work

**Comment by Banuk1616** [comment]
I've tried to recreate it, but I can't get it right either. But I'm not sure if I'm not doing something wrong.
